<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forgot</class>
 <widget class="QDialog" name="Forgot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Adresse_mail">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Adresse_mail_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nomut">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="nomut_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer adresse mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mdp_in">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nomut_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrez votre nouveau mot de passe</string>
   </property>
  </widget>
  <zorder>Adresse_mail</zorder>
  <zorder>Adresse_mail_2</zorder>
  <zorder>nomut</zorder>
  <zorder>nomut_2</zorder>
  <zorder>Submit</zorder>
  <zorder>mdp_in</zorder>
  <zorder>nomut_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
